roles/pulp3-devel/files/venv.bashrc

```diff
@@ -1,8 +0,0 @@
-# Set up virtualenvwrapper
```

Comment:
Not used, there's a template by the same name with the same contents that is being used.